include/gc.h

```diff
@@ -1363,8 +1363,12 @@ GC_API int GC_CALL GC_invoke_finalizers(void);
 /* The function is sometimes called keep_alive in other         */
 /* settings.                                                    */
 #if defined(__GNUC__) && !defined(__INTEL_COMPILER)
+#if 0 //Dejice commented out - compilation error
```

Comment:
I suggest we set a rule that "things that we've changed with the expectation that we'll change them again" should be something like `FIXME CHERI`. [The `FIXME` idiom is commonplace now, and the `CHERI` suffix allows us to differentiate our FIXMEs from any that boehm might already have now or in the future.]

Reply:
fixed in commit [408f667b](https://github.com/capablevms/bdwgc/pull/1/commits/408f667bb59d4d389e44ff70d5631bf922413ed9)